seedingscript.js

```diff
@@ -0,0 +1,187 @@
+        reviewText: Math.floor(Math.random() * 11) > 5 ? "'" + faker.lorem.paragraphs() + "'" : "'" + faker.lorem.paragraph() + "'",
+        isRecommended: Math.floor(Math.random() * 11) > 5 ? true : false,
+        commentCount: Math.floor(Math.random() * 11),
+        isHelpfulCount: Math.floor(Math.random() * 11),
+        isFunnyCount: Math.floor(Math.random() * 11),
+        isCleverAwardCount: Math.floor(Math.random() * 6),
+        warmBlanketAwardCount: Math.floor(Math.random() * 6),
+        saucyAwardCount: Math.floor(Math.random() * 6),
+        slowClapAwardCount: Math.floor(Math.random() * 6),
+        takeMyPointsAwardCount: Math.floor(Math.random() * 6),
+        wholesomeAwardCount: Math.floor(Math.random() * 6),
+        jesterAwardCount: Math.floor(Math.random() * 6),
+        fancyPantsAwardCount: Math.floor(Math.random() * 6),
+        whoaAwardCount: Math.floor(Math.random() * 6),
+        superStarAwardCount: Math.floor(Math.random() * 6),
+        wildAwardCount: Math.floor(Math.random() * 6),
+        treasureAwardCount: Math.floor(Math.random() * 6),
+        mindBlownAwardCount: Math.floor(Math.random() * 6),
+        goldenUnicornAwardCount: Math.floor(Math.random() * 6),
+        deepThoughtsAwardCount: Math.floor(Math.random() * 6),
+        heartWarmingAwardCount: Math.floor(Math.random() * 6),
+        hilariousAwardCount: Math.floor(Math.random() * 6),
+        hotTakeAwardCount: Math.floor(Math.random() * 6),
+        poetryAwardCount: Math.floor(Math.random() * 6),
+        extraHelpfulAwardCount: Math.floor(Math.random() * 6),
+        gottaHaveItAwardCount: Math.floor(Math.random() * 6),
+        michelangeloAwardCount: Math.floor(Math.random() * 6),
+        madScinetistAwardCount: Math.floor(Math.random() * 6),
+        isEarlyAccessReview: Math.floor(Math.random() * 11) > 5 ? true : false,
+        isPurchasedViaSteamKey: Math.floor(Math.random() * 11) > 5 ? true : false,
+        isActivatedViaSteamKey: Math.floor(Math.random() * 11) > 5 ? true : false,
+        userHoursOnRecordAtTimeOfReview: (Math.random() * userHoursOnRecord[userId]).toFixed(1),
+        reviewLanguage: Math.floor(Math.random() * 11) > 8 ? otherLanguages[Math.floor(Math.random() * otherLanguages.length)] : "'EN'",
+        reviewDate: `'${year}-${month}-${day} 00:00:00'`,
+      }
+
+      db.query(`INSERT INTO reviews (${Object.keys(data).join(', ')}) VALUES (${Object.values(data).join(', ')});`, (err) => {
+        if (err) {
+          console.log(`Error seeding review for gameId ${startingGameId} into table 'reviews': `, err);
+          return;
+        }
+      });
+      numReviews--;
+    }
+    startingGameId++;
+  }
+
+  console.log(`Reviews for all ${numGames}  seeded successfully into 'reviews' table`);
+}
+
+
+//----------------------------------------
+//Actual seeding
+//----------------------------------------
+
+
+let seedDatabase = () => {
+  seedUsers(252);
+  seedReviews(100, 252);
+};
+
+seedDatabase();
```

Comment:
Umm, I don't know where to offer improvements. I know Leslie said a PR review will very rarely be "Good job, you killed it!", but...
You separate logic from the actual seeding function to improve readability. You have taken the time to cover the details that aren't strictly necessary in your DB. I don't see any choppy code.
So, good job, you killed it lol